Gopkg.toml

```diff
@@ -40,7 +40,7 @@ required = [
 
 [[constraint]]
   branch = "master"
-  name = "github.com/giantswarm/e2e-harness"
+  name = "github.com/giantswarm/e2esetup"
```

Comment:
NICE

Reply:
Hold your horses :) We still have e2e-harness dependency here. But it's only one const.

Reply:
And whole `pkg/release` package which we are moving in scope of releng SWH...